### ISSUE TITLE
Cannot redefine property: $i18n

### ISSUE THREAD

**Issue by JoelPintoRibeiro** [panter/vue-i18next#65]
With the latest version of vue-i18next i have the same problem that is discussed here : https://github.com/kazupon/vue-i18n/issues/421

![image](https://user-images.githubusercontent.com/25738935/53888445-d3ec7480-4024-11e9-851b-247164cfd64d.png)

Is it possible to do the same fix for this plugin ? It's consist to check the property before we define it like this : 

```
if (!Vue.prototype.hasOwnProperty('$i18n')) {
  Object.defineProperty(Vue.prototype, '$i18n', {
    get () { return this._i18n }
  })
}
```

Thank you guys !

**Comment by JoelPintoRibeiro** [comment]
@claudiocro just a small notification !  did you have time to look at this issue? Thank you ;)

**Comment by JoelPintoRibeiro** [comment]
Is there anyone here ?
It's a good thing open source projects, but without listening to contributors it's useless ....

**Comment by rumbcam** [comment]
I've added this fix to my own pull request that also fixes another bug and makes the coverage go back up to 100% I'm hoping that the only reason this fix in #66 was ignored is because the coverage went down. If that is the case then with my pull request making the coverage go back up to 100% hopefully we can get a new release with this fixed
